prow/config/calico.md

```diff
@@ -0,0 +1,24 @@
+The calico.yaml file is from https://raw.githubusercontent.com/projectcalico/calico/v3.27.0/manifests/calico.yaml
+
+Once downloaded, run the following sed command to replace the default docker.io images with istio-testing's copies of them: 
+```shell
+sed -ie "s?docker.io?gcr.io/istio-testing?g" calico.yaml
+```
+
+In order to upgrade versions of calico we'll need to update the version below and then have someone with the ability to push run the following:
+```shell
+export VERSION=v3.27.0
+export REGISTRY=gcr.io/istio-testing
+
+docker pull calico/node:${VERSION}
```

Comment:
```
crane cp {docker.io,gcr.io/istio-testing}/calico/cni:v3.27.0
crane cp {docker.io,gcr.io/istio-testing}/calico/node:v3.27.0
crane cp {docker.io,gcr.io/istio-testing}/calico/kube-controllers:v3.27.0
```

Much better than pull+tag+push, faster and preserves the SHA

Reply:
Thanks, updated the doc